src/psyclone/tests/psyir/frontend/fparser2_test.py

```diff
@@ -794,6 +795,56 @@ def test_process_declarations():
             "interface" in str(error.value))
 
 
+@pytest.mark.usefixtures("f2008_parser")
+def test_declarations_with_initialisations(fortran_reader, fortran_writer):
+    '''Test that Fparser2Reader keeps all the variable initialisation
+    expressions, even thought some may end up in UnknownTypes for now.
```

Comment:
s/thought/though/ also, please clarify that that we get UnknownTypes because we don't yet support variable initialisation for anything that isn't a parameter.